README.md

```diff
@@ -2,6 +2,8 @@
 
 This website is built using [Docusaurus](https://docusaurus.io/), a modern static documentation generator written using React Native and is published to [https://embrace.io/docs/](https://embrace.io/docs/)
 
+Contributions are welcomed, no matter how small!
```

Comment:
@vitaliyf I'm gonna revert this commit on this branch as it conflicts with main now, looks like @davidlawrencer also updated the README so wasn't sure which should be taken

Reply:
Yea use his - I made this one to test deploys from your branch.

Reply:
<img width="859" alt="Screenshot 2024-09-18 at 1 29 41 PM" src="https://github.com/user-attachments/assets/50aa9537-843f-4356-be63-d0fca0709022">
@jpmunz can you clean up my typo at the end here 🙏

Reply:
@davidlawrencer I think best if you just open a separate PR for that, even if it's a small change helps keep things a bit more organized